### PR TITLE
[change] Skip commit check on commits prefixed with "Bump" #159

### LINKED ISSUE #159: [change] Skip commit check on commits prefixed with "Bump"
Pull requests created by dependabot fail the commit check: https://github.com/openwisp/openwisp-wifi-login-pages/pull/108
This makes reviewing these PRs timeconsuming because they all fail, also if the tests pass.

In the case of openwisp-wifi-login-pages, we have headless tests running, so if the build pass that should be a good indicator that we can merge.

### PR DESCRIPTION
**Pull request by Purhan**
Closes #159

### CODE REVIEW

File: openwisp-qa-check [hunk 1]
```diff
@@ -101,7 +101,7 @@ runblack() {
 
 runcheckcommit() {
   if [ -z "$COMMIT_MESSAGE" ]; then COMMIT_MESSAGE=$(git log -1 --pretty=%B); fi
-  if [ "$TRAVIS" = true ] && [ "$TRAVIS_PULL_REQUEST" = false ]; then
+  if [[ "$COMMIT_MESSAGE" =~ ^Bump ]] || ([ "$TRAVIS" = true ] && [ "$TRAVIS_PULL_REQUEST" = false ]); then
```
Comment (nemesisdesign) [inline openwisp-qa-check:104]: look here: https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/qa.py#L98
That's where we should do it, please also add a test for it.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Thanks for taking this @Purhan, we should use a different approach though, see my comment below.

File: openwisp-qa-check (hunk 1 above)
Comment (Purhan) [reply openwisp-qa-check]: Thanks for reviewing, @nemesisdesign please take a look now.

**Review by nemesisdesign — APPROVED** [review]
Perfect, thanks!

**Pull request merged**